GET.md

```diff
@@ -49,8 +49,20 @@ maintainers:
 features:
  - dco_check
  - comments
+ - commit_linting
 ```
 
+Features:
+
+* `dco_check` - checks that each commit finishes with a "Signed-off-by:" statement
+* `comments` - allows `maintainers` to issue commands to Derek to add labels etc
+* `commit_linting` - applies linting rules to commit messages
+
+Commit linting ensures:
+
+* subjects start with a capital letter
+* subject lines are less than 50 characters
```

Comment:
Less than or equal to 50 characters

Reply:
GitHub will allow up to 72 characters before it wraps. I'm wondering whether we should expand the limit?

Reply:
In all areas that the subject is displayed? Is the client similar? The referenced article is 4 years old so gh may have changed in that time.

I liked the idea of that being configurable, even if it was through env-vars.

Reply:
I could move the hard-limit to 72 characters as per GitHub UI

Reply:
`subject lines contain no more than 72 characters`